docs/src/Toast.doc.js

```diff
@@ -51,145 +50,164 @@ card(
-            paddingX={1}
-            position="fixed"
-          >
-            <Toast
-              icon="arrow-circle-forward"
-              text="Same great profile, just a new look. Learn more?"
-            />
-          </Box>
+      <Button
+        inline
+        text={ this.state.showGuideToast ? 'Close toast' : 'Show guide toast' }
+        onClick={this.handleGuideClick}
+        size='md'
+      />
+      <Box
+        fit
+        dangerouslySetInlineStyle={{
+          __style: {
+            bottom: 150,
+            left: '50%',
+            transform: 'translateX(-50%)',
+          },
+        }}
+        paddingX={1}
+        position='fixed'
+      >
+        {this.state.showGuideToast ? (
+          <Toast
+            icon='arrow-circle-forward'
+            text='Same great profile, just a new look. Learn more?'
+          />
         ) : null}
+      </Box>
       </div>
-    )}
+    );
+  }
+}`}
   />
 );
 
 card(
-  <StateRecorder
-    description={`
-    You can use Toasts to indicate that something wrong occurred by setting the color to orange.
-
-    ~~~jsx
-    <Toast color="orange" text="Oops, we couldn't find that!" />
-    ~~~
-  `}
+  <Example
+    description="
+      You can use Toasts to indicate that something wrong occurred by setting the color to orange.
+    "
     name="Error Toasts"
-    fn={atom => (
+    defaultCode={`
+class ToastExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+      showErrorToast: false
+    };
+    this.handleErrorClick = this.handleErrorClick.bind(this);
+  }
+  handleErrorClick({ event }) {
+    this.setState(prevState => ({ showErrorToast: !prevState.showErrorToast }));
+  };
+  render() {
+    return (
       <div>
-        <Button
-          inline
-          text={atom.deref().showToast ? 'Close toast' : 'Show error toast'}
-          onClick={() => atom.reset({ showToast: !atom.deref().showToast })}
-          size="md"
-        />
-        {atom.deref().showToast ? (
-          <Box
-            fit
-            dangerouslySetInlineStyle={{
-              __style: {
-                bottom: 68,
-                left: '50%',
-                transform: 'translateX(-50%)',
-              },
-            }}
-            paddingX={1}
-            position="fixed"
-          >
-            <Toast color="orange" text="Oops, we couldn't find that!" />
-          </Box>
+      <Button
+        inline
+        text={ this.state.showErrorToast ? 'Close toast' : 'Show error toast' }
+        onClick={this.handleErrorClick}
+        size='md'
+      />
+      <Box
+        fit
+        dangerouslySetInlineStyle={{
+          __style: {
+            bottom: 250,
+            left: '50%',
+            transform: 'translateX(-50%)',
+          },
+        }}
+        paddingX={1}
+        position='fixed'
+      >
+        {this.state.showErrorToast ? (
+          <Toast color='orange' text='Oops, we couldn't find that!' />
```

Comment:
@nithindavid Thank you so much for adding these live docs!

Seems like this line results in an error since the `'` in `couldn't` is not escaped. We could probably change this to use double quotes instead `text="Oops, we couldn't find that!"`.

![image](https://user-images.githubusercontent.com/127199/37660091-d62f1a20-2c0e-11e8-9dd5-8e0ac166a3c9.png)

Reply:
@christianvuerings  Done 👍